Makefile

```diff
@@ -110,11 +110,3 @@ scan-vulnerability:
 	mkdir -p ./tmp/clair/usabillabv
 	cat ./tmp/build-*.tags | xargs -I % sh -c 'clair-scanner --ip 172.17.0.1 -r "./tmp/clair/%.json" -l ./tmp/clair/clair.log % || echo "% is vulnerable"'
 	docker-compose -f test/security/docker-compose.yml -p clair-ci down
-
-ci-scan-vulnerability:
```

Comment:
the commit says "drop Circle CI", but drops Clair checking? will this step be migrated to GHA?

Reply:
We had two `scan-vulnerability`. This one that we are dropping is called `ci-scan-vulnerability`, which was being used by CircleCI.